pages/general-information-and-resources/tech-policies/tts-pages.md

```diff
@@ -0,0 +1,84 @@
+- [ ] Confirm your Website on [Touchpoint's Registry](https://touchpoints.app.cloud.gov/admin/websites) or Submit a new website request in [Touchpoints](https://touchpoints.app.cloud.gov/admin/websites/new)
+
+## Authority to Use (ATU) Review
+Submit an [Authority to Use (ATU) Request](https://github.com/GSA-TTS/tts-pages/issues/new?)
+
+- [ ] Resolve any Critical or High security findings from Security Scanners
+- [ ] Resolve any Web Scan findings provided by GSA Security Team
+
+Once the review is completed the Site Manager will be sent their ATU Approval package via Docusign. The Site Manager will be responsible for managing Security Findings over the lifecycle of the Website.
+
+## Maintaining Approved Sites 
+Sites hosted on TTS Pages are required to have their URLs scanned in accordance with CIO-IT Security-06-30: Managing Enterprise Cybersecurity Risk and GSA’s parameter for National Institute of Standards and Technology (NIST) Special Publication (SP) 800-53, control RA-5, Vulnerability Scanning. 
+
+>This is performed after the ATU request is submitted
+
+## 	Reassessment 
+A Site’s ATU will have to be reassessed and an ATU reissued if the Site is found `NOT` to be in conformity with the requirements within this guide.
+
+Conditions/events that may require a reassessment of the ATU include:
+- New third party integrations not on the approved list are added.
+- The data types or information presented on the site changes.
+- A significant security incident occurs.
+- There are deviations from the ATU maintenance requirements.
+
+>This Determination is made by the TTS Pages System Owner `tts-tech-operations@gsa.gov`. Generally, this is done if security findings are not being addressed promptly.
+
+Website Managers will be notified, the following steps are only in the event that the Website Manager is none responsive.
+
+## Failure to Maintain Site - Site Removal
+Sites that fail to maintain the ATU requirements will be issued a formal notice. The TTS Pages team may take steps to disable the site or remediate the vulnerabilities. ATU site owners who hit certain triggers of overdue POA&Ms and/or failure to maintain alignment to ATU requirements will be required to provide a **Corrective Action Plan (CAP)** addressing the plan to address the deficiencies. 
+
+The CAP must be approved by the Site owner, System Owner, ISSM, and IST Director. Sites or Site owners who fail to respond to a CAP, or complete approved actions will be removed from the ATO boundary, and will no longer be authorized. The removal process steps are further described below: 
+- **Detailed Finding Review (DFR)** - Site owners will be issued a DFR upon failing to address a deficiency within the site or alignment with the ATU requirements.
+- **Corrective Action Plan** - Site Owners who fail to adequately respond or address a DFR, will be issued a CAP request. 
```

Comment:
Site Owner is capitalized here, but not "Site owners" in the bullet point above. I still think this is easier to read without the Extra Capitals, but whichever, please be consistent.

Reply:
There's a lot of passive voice throughout this document. Site owners _will be issued a DFR_... _will be issued a request_... etc. It'd be clearer and easier to read throughout if we start with _who_ is issuing these to the site owners. "The TTS Pages office will issue a DFR in the event a site owner fails to adequately respond.... " etc. It also makes it clearer who is doing what, rather than things just happening.

If you'd like, I can take a stab at reordering these phrases. I don't mind :)

Reply:
Updated to avoid passive language